File: gaia-sdk-javascript/src/graphql/response/type/OnDeleted.ts


import {Uuid, ISO8601, Struct} from "../../GaiaFunctionClient";
import {RuntimeState} from "../../request/enumeration/RuntimeState";
import {SkillState} from "../../request/enumeration/SkillState";

export interface OnDeleted {
    id?:Uuid,
    identityId?:Uuid,
    reference?:Uuid,
    type?:string
}

File: gaia-sdk-java/gaia-sdk-graphql/src/main/kotlin/gaia/sdk/response/type/OnDeleted.kt
package gaia.sdk.response.type

import gaia.sdk.Uuid
import gaia.sdk.ISO8601
import gaia.sdk.Struct
import gaia.sdk.response.intf.*
import com.fasterxml.jackson.annotation.JsonProperty
import com.fasterxml.jackson.annotation.JsonCreator
import gaia.sdk.request.enumeration.*

data class OnDeleted @JsonCreator constructor(
    @JsonProperty("id") val id:Uuid? = null, 
    @JsonProperty("identityId") val identityId:Uuid? = null, 
    @JsonProperty("reference") val reference:Uuid? = null, 
    @JsonProperty("type") val type:String? = null
)